tests/speculation/__init__.py

```diff
@@ -0,0 +1,29 @@
+#  Copyright 2025 Google LLC.
+#  Copyright (c) Microsoft Corporation.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import asyncio
+
+from test_helpers import wait_for_event
+
+
+async def wait_for_prefetch_event_with_timeout(websocket,

```

Comment:
(optional) you can mark it as fixture, so there will be no need in passing `websocket`.